client/state/reader/sites/test/reducer.js

```diff
@@ -101,109 +114,127 @@ describe( 'reducer', ( ) => {
 				title: 'example!',
 				options: {
 					is_redirect: true,
-					unmapped_url: 'http://formerlyexample.com/foo/bar'
-				}
+					unmapped_url: 'http://formerlyexample.com/foo/bar',
+				},
 			} );
 		} );
 
-		it( 'should serialize site entries', ( ) => {
+		it( 'should serialize site entries', () => {
 			const unvalidatedObject = deepFreeze( { hi: 'there' } );
 			expect( items( unvalidatedObject, { type: SERIALIZE } ) ).to.deep.equal( unvalidatedObject );
 		} );
 
-		it( 'should not serialize errors', ( ) => {
+		it( 'should not serialize errors', () => {
 			const stateWithErrors = deepFreeze( {
 				12: { ID: 12, name: 'yes' },
 				666: {
 					ID: 666,
-					is_error: true
-				}
+					is_error: true,
+				},
+			} );
+			expect( items( stateWithErrors, { type: SERIALIZE } ) ).to.deep.equal( {
+				12: { ID: 12, name: 'yes' },
 			} );
-			expect( items( stateWithErrors, { type: SERIALIZE } ) ).to.deep.equal( { 12: { ID: 12, name: 'yes' } } );
 		} );
 
-		it( 'should reject deserializing entries it cannot validate', sinon.test( function() {
-			const unvalidatedObject = deepFreeze( { hi: 'there' } );
-			this.stub( console, 'warn' ); // stub warn to suppress the warning that validation failure emits
-			expect( items( unvalidatedObject, { type: DESERIALIZE } ) ).to.deep.equal( {} );
-		} ) );
+		it(
+			'should reject deserializing entries it cannot validate',
+			sinon.test( function() {
+				const unvalidatedObject = deepFreeze( { hi: 'there' } );
+				this.stub( console, 'warn' ); // stub warn to suppress the warning that validation failure emits
+				expect( items( unvalidatedObject, { type: DESERIALIZE } ) ).to.deep.equal( {} );
+			} )
+		);
 
-		it( 'should deserialize good things', ( ) => {
+		it( 'should deserialize good things', () => {
 			const validState = deepFreeze( {
 				1234: {
 					ID: 1234,
-					name: 'Example Dot Com'
-				}
+					name: 'Example Dot Com',
+				},
 			} );
 			expect( items( validState, { type: DESERIALIZE } ) ).to.deep.equal( validState );
 		} );
 
-		it( 'should stash an error object in the map if the request fails', ( ) => {
-			expect( items( {}, {
-				type: READER_SITE_REQUEST_FAILURE,
-				error: new Error( 'request failed' ),
-				payload: { ID: 666 }
-			} ) ).to.deep.equal( { 666: { ID: 666, is_error: true } } );
+		it( 'should stash an error object in the map if the request fails', () => {
+			expect(
```

Comment:
so verbose.
I don't mind the previous style here, but the new one isn't bad either